sound/soc/sof/sof-priv.h

```diff
@@ -585,8 +585,6 @@ int snd_sof_ipc_set_get_comp_data(struct snd_sof_ipc *ipc,
  * There is no snd_sof_free_topology since topology components will
```

Comment:
@ranj063 some of the patches are independent of this PR and I think we can already merge them to decrease the number of patches in the PR and to avoid re-reviewing them all the time.

Reply:
yes, I'd really like a 'prepare multi-client' PR so that we can decrease the volume of code. this is what we did for S0ix and it was much easier. Anything about 10 commits is just awful on GitHub, and even in emails it's borderline manageable.